skype/skype-ps/skype/Remove-CsTeamsAudioConferencingPolicy.md

```diff
@@ -0,0 +1,111 @@
+---
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/en-us/powershell/module/skype/remove-csteamsaudioconferencingpolicy
+schema: 2.0.0
+---
+
+# Remove-CsTeamsAudioConferencingPolicy
+
+## SYNOPSIS
+
+Deletes a custom Teams audio conferencing policy. Audio conferencing policies are used to manage audio conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization.
+
+## SYNTAX
+
+```
+Remove-CsTeamsAudioConferencingPolicy [-Identity] <String> [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Deletes a previously created TeamsAudioConferencingPolicy. Any users with no explicitly assigned policies will then fall back to the default (Global) policy in the organization. You cannot delete the global policy from the organization.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+PS C:\> Remove-CsTeamsAudioCOnferencingPolicy -Identity “Emea Users”
+```
+
+In the example shown above, the command will delete the “Emea Users” audio conferencing policy from the organization's list of policies.
+
+## PARAMETERS
+
+### -Force
+
```

Comment:
Please add description.

Reply:
"Suppresses any confirmation prompts that would otherwise be displayed before making changes."